jenkins/bin/reload-jobs.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# create and update all jobs
+set -e
+
+reload() {
+
+    # the job-reloading-job needs to update itself, but note that changes
+    # won't be picked up in this pass because we're already running the
+    # job. kick the build off manually if you need it immediately, otherwise
+    # wait until the next pass
+    xmlstarlet \
+        ed \
+        -u '//project/scm/userRemoteConfigs/hudson.plugins.git.UserRemoteConfig/url' \
+        -v "${GITHUB_JOBS_REPO}" \
+        -u '//project/scm/branches/hudson.plugins.git.BranchSpec/name' \
+        -v "${GITHUB_JOBS_SPEC:-master}" \
+        /usr/share/jenkins/templates/jenkins-jobs.config.xml \
+        > "${JENKINS_HOME}/jobs/jenkins-jobs/config.xml"
+
+    for job in ${WORKSPACE}/jenkins/jobs/*; do
+
+        local jobname=$(basename ${job})
+        local config=${WORKSPACE}/jenkins/jobs/${jobname}/config.xml
```

Comment:
Also, got tripped by the newly added `jenkins` level in this path. My `jobs` repo has `jobs` at the root. All figured for now.

Would be great to have an ENV var to define the exact path to jobs. Or I guess, people could rewrite their own version of this script.